resolwe_bio/tests/processes/test_support_processors.py

```diff
@@ -216,3 +216,18 @@ def test_library_strandedness(self):
         lib_strandedness_paired = self.run_process('library-strandedness', paired_input)
         self.assertFields(lib_strandedness_paired, 'strandedness', 'IU')
         self.assertFields(lib_strandedness_paired, 'fragment_ratio', 1.0)
+
+    @tag_process('prepare-annotation-dexseq')
+    def test_prepare_dexseq_gtf(self):
+        with self.preparation_stage():
+            inputs = {
+                'src': 'annotation_ensembl_mus_musculus_short.gtf.gz',
+                'source': 'ENSEMBL',
+                'species': 'Mus musculus',
+                'build': 'GRCm38.p5',
+            }
+            annotation = self.run_process('upload-gtf', inputs)
+
+        inputs = {'annotation': annotation.pk}
+        prepare_dexseq = self.run_process('prepare-annotation-dexseq', inputs)
+        self.assertFileExists(prepare_dexseq, 'annot')
```

Comment:
Why isn't hash checked here? Annotation files don't contain floats.

Reply:
The third-party script that is used to produce the annotation file returns a different result every time. So the output annotation is different every time. That does not affect further analysis however as what is different is only the order of the gene names in a single row (separated by '+') and not the order of rows.